pkgs/development/python-modules/hachoir/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, urwid
+, wxPython_4_0
+, six
+, flake8
```

Comment:
I don't know Nix. Is this a build dependency, or a runtime dependency? flake8 is not needed to run Hachoir.

Reply:
This part should only import flake8. It is listed as a dependency for running tests in `checkInputs`, which means it is not a runtime dependency, only buildtime.